rank/services/test.ts

```diff
@@ -29,26 +25,48 @@ type Props = {
 async function service({
   namespace,
   testId,
+  env,
```

Comment:
What I was picturing is evatually having all this information stored in the search template so you don't need to have all these floating values everywhere. This is OK for now though.

Reply:
yeah, i agree tbh - think we're both on the same page about where all this data should be going eventually